Documentation/WritingDocsOfficial/HowYouCanHelp.rst

```diff
@@ -147,26 +145,14 @@ issue for it, if you cannot make the changes yourself.
    in a manual. Please focus your
    efforts mostly on the latest `main` branch, to get that up to date and ready!
 
-You can for example start with:
-
-*  :ref:`t3extbasebook:start` : This currently needs some work in all branches. Check if the text
-   is up to date and technically correct. Also, the text needs to be shortened in some parts and
-   language improvements are necessary (grammar, style). For the `main` branch, you can
-   `check off this review list <https://github.com/TYPO3-Documentation/TYPO3CMS-Book-ExtbaseFluid/issues/296>`__
-   once you have started reviewing. This makes it easier for other reviewers to just skip the parts
-   that are already ok.
-*  :ref:`t3coreapi:start`: The same applies as for :ref:`t3extbasebook:start` except that
-   "TYPO3 Explained" is generally in much better shape. Still, you can help by reviewing and
-   `checking off this review list <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/537>`__
-
 .. tip::
 
    If you don't have write access to a GitHub repository (which is usually the case if you
    just started), you cannot check off the checkmarks in the review list. In that case, just leave a comment
    in the issue.
 
 Once you start and are confident that you can make progress, you may want to talk about
-your intentions in the #typo3-documentation Slack channel (as this is a bigger change).
+your intentions in the `#typo3-documentation` Slack channel (as this is a bigger change).
```

Comment:
I suggest to link the channel for easy access. Otherwise the user has to search for the channel on their own.

Reply:
how to link a channel?

Reply:
https://typo3.slack.com/app_redirect?channel=C028JEPJL